wayshot/src/ext_wayshot.rs

```diff
@@ -0,0 +1,366 @@
+use image::{GenericImageView, ImageEncoder, ImageError};
+use std::{env, fs, path::PathBuf};
```

Comment:
no.. do not copy this part.. wayshot does not use notify_rust.. www.. maybe you should read carefully about the repo